### PR TITLE
#188 [华农兄弟] 新毙鼠山庄

### PR DESCRIPTION
**Pull request by vincenthychow**
#188 [华农兄弟] 新毙鼠山庄 - https://youtu.be/2aIaAFJx0RM

### CODE REVIEW

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+#我們再去看一下其它地方
+Let's take a look at other places.
+
+0:02:04.720,0:02:05.560
+#這裡還有做哦
+Some cells are being built here too.
+
+0:02:07.600,0:02:12.040
+#這一邊  就這樣一過去應該有800平方左右
+This side, from, from here, all the way, it should be around 800 meter square.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
This side should be around 800 square meters all the way.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+
+0:02:12.420,0:02:15.040
+#加上樓上800有1600  有1600平方
+Together with 800 up stair, it means 1600, 1600 meter square.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Together with 800 upstairs, it adds up to 1600 square meters.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+
+0:02:15.640,0:02:20.360
+#這裡空地的話1700多平方  朼將近3畝  3畝地
+This empty space is more than 1700 meter square, which is almost 3 "mu", 3 "mu" of land. (1 "mu" = 666.666 meter square)
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
This empty space occupies more than 1700 square meters, which is almost 3 "mu", 3 "mu" of land. (1 "mu" = 666.666 square meters)
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 4]
```diff
@@ -0,0 +1,481 @@
...
+#種一點象草
+and plant some elephant grass,
+
+0:02:24.360,0:02:25.420
+#種點像草就可以啦
+some elephant grass, then it would be nice.
+
+0:02:28.000,0:02:29.100
+#嗯  我去樓上看一下
+Um. Let me go up stair to have a look.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Um. Let me go upstairs to have a look.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 5]
```diff
@@ -0,0 +1,481 @@
...
+
+0:02:29.400,0:02:32.020
+#還上還有  也很 很大的哦  樓上
+There is still, very, it's very large, up stair.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
There is still, very, it's very large, upstairs.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 6]
```diff
@@ -0,0 +1,481 @@
...
+
+0:02:37.420,0:02:40.080
+#你看  這裡是住人的哦  都是一隔隔開的
+Look! This is a residential area. It's divided into units.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Look! This is a residential area. It's divided into compartments.
```

Comment (WuStevenShengyang) [reply subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]: 这里会不会compartment比unit好一些

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 7]
```diff
@@ -0,0 +1,481 @@
...
+#前面看一下  前面有點黑喲
+Let me have a look there. It's a bit dark.
+
+0:02:45.140,0:02:46.500
+#還是鎖匙的
+I need a key.
+
+0:02:49.140,0:02:50.720
+#哇  這還有間套房啊  (其實視頻顯示不是套房，但不理了，照譯)
+Wa! Here is suite.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Wow! Here is a suite.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 8]
```diff
@@ -0,0 +1,481 @@
...
+
+0:02:53.640,0:02:56.020
+#這裡  還帶花的  不知道是誰的
+Here, with the flowery pattern, whose' that?
+
+0:02:57.960,0:02:59.320
+#哇  這是好東西咬[SIC]喲
+Wa! This is wonderful!
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Wow! This is wonderful!
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 9]
```diff
@@ -0,0 +1,481 @@
...
+#下次我要把它吃啦
+I'll eat them next time.
+
+0:03:10.820,0:03:12.440
+#你看  這兩三個
+Here, a couple of,
+
+0:03:12.520,0:03:14.260
+#腌  腌那個咸菜的缸哦  (口吃出現地方是句子開頭，所以保留位置，但字改變了)
+jars, jars for pickling vegetables.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
jars, jars for pickled vegetables.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 10]
```diff
@@ -0,0 +1,481 @@
...
+
+0:03:14.740,0:03:16.820
+#到時候我們就有  咸菜吃啦
+We can have, have pickles.
+
+0:03:20.960,0:03:22.540
+#哇  這好多房間啊
+Wa! A lot of rooms.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Wow! A lot of rooms.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 11]
```diff
@@ -0,0 +1,481 @@
...
+#下次我們有空的時候就在那裡釣魚
+Next time when we have the time, we can go fishing there.
+
+0:06:09.620,0:06:12.020
+#你看  魚竿都不用  魚竿都不用帶
+Look! No rod, you don't even need a rod.
+
+0:06:12.280,0:06:13.120
+#上面就是竹子
+Above the pond is already a lot of bamboo.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Above the pond are bamboos.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 12]
```diff
@@ -0,0 +1,481 @@
...
+#下次我抽完水來抓
+I'll catch it after draining out the water next time.
+
+0:06:01.300,0:06:02.460
+#那裡還有一個大水潭哦
+There's another big pond.
+
+0:06:03.720,0:06:05.380
+#額  前兩天我還看了一下
+Eh, I had a look a couple of days ago.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Eh, I looked into it a couple days ago.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 13]
```diff
@@ -0,0 +1,481 @@
...
+#邊是蓋起來  不蓋的話  煮魚湯一樣那麼熱
+The edge needs to be covered. Otherwise, it would be as hot as making a soup,
+
+0:05:45.340,0:05:46.360
+#把魚都煮死啦
+and the fish would be boiled to death.
+
+0:05:48.420,0:05:50.720
+#你看  這下面有3口魚塘哦
+Look? Down there are three ponds.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Look! Down there are three ponds.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 14]
```diff
@@ -0,0 +1,481 @@
...
+#嗯
+Um?
+
+0:04:52.060,0:04:52.940
+#誰家的鴨子
+Whose duck is it?
+
+0:05:29.900,0:05:31.220
+#哇  這還有兩個池哦
+Wa! Here are two little ponds too.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Wow! Here are two little ponds too.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 15]
```diff
@@ -0,0 +1,481 @@
...
+#新的哦  這個椅子  就是有點髒  抺乾淨就行了
+They are new, just a bit dirty. Should be fine after some cleaning.
+
+0:03:39.500,0:03:40.460
+#玻璃有點爛
+The glass is a bit broken.
+
+0:03:41.600,0:03:42.640
+#哇  這是
+Wa! This is
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Wow! This is
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv [hunk 16]
```diff
@@ -0,0 +1,481 @@
...
+#這個  這個
+This one, this one
+
+0:03:26.340,0:03:27.320
+#這個是辦公室
+This is an office.
+
+0:03:28.580,0:03:30.120
+#哇  還有兩張椅子在這裡
+Wa! There are even two chairs.
```
Comment (WuStevenShengyang) [inline subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
Wow! There are even two chairs.
```

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv (hunk 1 above)
Comment (immoonancient) [reply subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]:
```suggestion
This side, from, from here, all the way, it should be around 800 square meters.
```
语言风格是故意为之的，保留
https://simple.wikipedia.org/wiki/Square_metre:
> A "square metre" is not the same thing as a "metre square" [2] - although it is true that exactly 1 square metre is exactly 1 meter square. But that only works for the numbers 1 and 0; no others.

File: subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv (hunk 9 above)
Comment (immoonancient) [reply subtitles/hua-nong-brothers/20190914-new-concentration-camp.sbv]: 用 pickling 应该没问题，表示这里强调动作

**Pull request merged**